client/components/Client.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+const Client = (props) => {
+  console.log('[Client.jsx] clientData', props.clientData)
+
+  const clientData = Object.entries(props.clientData).map( (field, i) => {
```

Comment:
This will fail linting - need single quotes, not spaces when destructuring, semicolons